vue-components/src/components/Button.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+	<button :class="[ 'wikit', 'wikit-Button', `wikit-Button--${ flavor }` ]">
+		<slot />
+	</button>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+
+/**
+ * A Button
+ */
+export default Vue.extend( {
+	name: 'Button',
+	props: {
+		/**
+		 * The flavor of the button.
+		 * One of: neutral
+		 */
+		flavor: {
+			type: String,
+			validator( value: string ): boolean {
+				return [ 'neutral' ].includes( value );
+			},
+			default: 'neutral',
+		},
+	},
+} );
+</script>
+
+<style lang="scss">
+/**
+ * Construction site identified:
+ * How do we name the component level token?
+ *
+ * Suggestion for naming scheme:
+ * wikit-<Component>-[prop]-[state]-css-property
```

Comment:
This should be documented somewhere. ADR is too big but "Developing Components" in the documentation would lend itself.